hubspot/serializers.py

```diff
@@ -6,20 +6,29 @@
 from ecommerce.models import CouponVersion, ProductVersion, CouponRedemption
 
 
+ORDER_STATUS_MAPPING = {
+    models.Order.FULFILLED: "processed",
+    models.Order.FAILED: "checkout_completed",
+    models.Order.CREATED: "checkout_completed",
+    models.Order.REFUNDED: "processed",
```

Comment:
@pdpinch or @Ferdi should this be "cancelled" instead?  edX doesn't seem to have a refund mapping, but "cancelled" is an available status for Hubspot.